First draft of 21.0.0.8-beta post (#1524)
OpenLiberty/blogs: posts/2021-07-08-beta-21008-beta.adoc

---
layout: post

> title: "In Open Liberty 21.0.0.8-beta a number of new features and improvements including; new logging format (TBASIC), The Stale Connection Identification, MicroProfile Health 3.1, Jakarta EE 9.1 convenience features and MicroProfile 4.1 improvements"

Reviewer: ```suggestion
title: "In Open Liberty 21.0.0.8-beta: MicoProfile 4.1 with MicroProfile Health 3.1; a new logging format (TBASIC); Stale Connection Identification; Jakarta EE 9.1 convenience features"
```
